### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by queenypingcap**
Resolving conflicts...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 
 - __Geo-Replication__  
-TiKV uses Raft and placement driver to support Geo-Replication. 
+TiKV uses Raft and Placement Driver to support Geo-Replication. 
 
 - __Horizontal scalability__  
-With carefully designed Raft groups and placement driver, TiKV excels in horizontal scalability and can easily scale up to hold 100s of TBs of data.
+With Placement Driver and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale up to hold 100s of TBs of data.
```
Comment (shenli) [inline README.md]: scale up -> scale or scale out
"scale up" is not commonly used.

File: README.md [hunk 2]
```diff
@@ -30,15 +30,12 @@ This figure represents tikv-server software stack.
...
-- Node: A physical node in cluster. Node id must be unique in global.
-- Store: A node has one or some stores. Generally a store involves one disk. Each store maps to different paths, and store id must be unique in global either. Multiple stores primarily support a plurality of disks in one node.
-- Region: Region is a logical concept. Key-Value datas are grouped by region. Region is the smallest unit of data movement, that's geographic-replication unit. Every region is supported by a raft group and region id must be unique in global. 
-- Peer: Peer is a logical concept. It stands for a raft-worked participant in a raft group. A peer in raft-worked group maybe turn up three roles, which are candidate, leader and follower.
+- Placement driver: Placement Driver (PD) is the cluster manager of TiKV. PD periodically checks replication constraints to balance load and data automatically.
+- Node: A physical node in the cluster. Within each node, there are one or more Stores. Within each Store, there are many Regions.
+- Store: There is a RockDB within each Store and it stores data into local disk.
```
Comment (shenli) [inline README.md]: Swap Store and Node. Because in the definition of Node, it mentions Store.

File: README.md (hunk 1 above)
Comment (c4pt0r) [inline README.md]: Or just '...can easily scale to 100+ TBs of data'

File: README.md [hunk 3]
```diff
@@ -30,15 +30,12 @@ This figure represents tikv-server software stack.
...
+- Region: Region is the basic unit of key-value data movement. Each Region is replicated to multiple Nodes. These multiple replicas forms a Raft group.
```
Comment (disksing) [inline README.md]: Use `key-value` here but `Key-Value` at L7. I suppose they should be unified.

File: README.md [hunk 4]
```diff
@@ -4,13 +4,13 @@
...
+TiKV is a distributed Key-Value database which is based on the design of Google Spanner and HBase, but it is much simpler without dependency on any distributed file system. With the implementation of the Raft consensus algorithm in Rust and consensus state storing in RocksDB, it guarantees data consistency. Placement Driver which is introduced to implement sharding enables automatic data migration. The transaction model is similar to Google's Percolator with some performance improvements. TiKV also provides snapshot isolation (SI), serializable snapshot isolation (SSI), and externally consistent reads and writes in distributed transactions. See [Tikv-server software stack](#tikv-server-software-stack) for more information. TiKV has the following primary features:
```
Comment (siddontang) [inline README.md]: /storing in RocksDB/stored in RocksDB/s ?

File: README.md (hunk 3 above)
Comment (siddontang) [inline README.md]: form a Raft group.

**Pull request merged**